package.json

```diff
@@ -1,93 +1,104 @@
-    "lint:changelog": "auto-changelog validate --prettier",
-    "lint:constraints": "yarn constraints",
-    "lint:dependencies": "depcheck && yarn dedupe",
-    "lint:eslint": "eslint . --cache --ext js,cjs,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:constraints --fix && yarn lint:misc --write && yarn lint:dependencies && yarn lint:changelog",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "build": "yarn ts-bridge --project tsconfig.build.json --verbose",
+    "build:clean": "rimraf dist '**/*.tsbuildinfo' && yarn build",
+    "build:docs": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run build:docs",
+    "build:types": "tsc --build tsconfig.build.json --verbose",
+    "changelog:update": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:update",
+    "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
+    "create-package": "ts-node scripts/create-package",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn constraints && yarn lint:dependencies",
+    "lint:dependencies": "depcheck && yarn dedupe --check",
+    "lint:dependencies:fix": "depcheck && yarn dedupe",
+    "lint:eslint": "eslint . --cache --ext js,cjs,mjs,ts",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix && yarn lint:dependencies:fix",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' '!merged-packages/**' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
-    "test": "jest && jest-it-up && attw --pack",
-    "test:watch": "jest --watch"
+    "prepare-preview-builds": "./scripts/prepare-preview-builds.sh",
+    "publish-previews": "yarn workspaces foreach --all --no-private --parallel --verbose run publish:preview",
+    "setup": "yarn install",
+    "test": "yarn test:scripts --silent --collectCoverage=false --reporters=jest-silent-reporter && yarn test:packages",
+    "test:clean": "yarn workspaces foreach --all --parallel --verbose run test:clean && yarn test",
+    "test:packages": "yarn test:verbose --silent --collectCoverage=false --reporters=jest-silent-reporter",
+    "test:scripts": "yarn jest --config ./jest.config.scripts.js --silent",
+    "test:verbose": "yarn workspaces foreach --all --parallel --verbose run test:verbose",
+    "update-readme-content": "ts-node scripts/update-readme-content.ts",
+    "workspaces:list-versions": "./scripts/list-workspace-versions.sh"
+  },
+  "simple-git-hooks": {
+    "pre-push": "yarn lint"
+  },
+  "resolutions": {
+    "elliptic@6.5.4": "^6.5.7",
+    "fast-xml-parser@^4.3.4": "^4.4.1",
+    "ws@7.4.6": "^7.5.10"
   },
   "devDependencies": {
-    "@arethetypeswrong/cli": "^0.15.3",
+    "@babel/core": "^7.23.5",
```

Comment:
We can be certain that all these dependencies are needed because we use `depcheck` in our linting scripts, which ensures that all dependencies are being used.

<img width="570" alt="Screenshot 2024-10-07 at 8 30 52 PM" src="https://github.com/user-attachments/assets/a0c10e38-4fab-442d-b2b5-89aa20f942a9">